networkConfig.ts

```diff
@@ -21,6 +22,9 @@ const local: NetworkConfig = {
   allowedTokenList: {
     ETH: ethers.constants.AddressZero,
   },
+  minTransactionAmount: {
```

Comment:
use the linked config: 
`allowedTokenList: { address: `0x${string}`,  minTransactionAmount: string}`